test/locally-installed-sdk.spec.js

```diff
@@ -286,6 +286,33 @@ describe('Locally Installed Google DataStore Emulator Test', () => {
       })
   });
 
+  it('should set environment variables', () => {
+    process.env.GCLOUD_PROJECT = 'my-test-GCLOUD_PROJECT';
+    let options = {
+      debug: true,
+      port: 8081,
+    };
+
+    let emulator = new Emulator(options);
+
+    return emulator.start()
+      .then(() => {
+        expect(process.env.DATASTORE_EMULATOR_HOST).to.be.equal(`localhost:8081`);
+        expect(process.env.DATASTORE_EMULATOR_HOST_PATH).to.be.equal(`localhost:8081/datastore`);
+        expect(process.env.DATASTORE_HOST).to.be.equal(`http://localhost:8081`);
+        // shouldn't this be the same as GCLOUD_PROJECT, but they don't
+        // see: https://gist.github.com/mdornseif/d9bd0c55a7aab298d0a29bee124302c8
+        expect(process.env.DATASTORE_DATASET).to.be.equal(`test`);
```

Comment:
It fails because the default value of the `options.project` is test. I checked the documentation and it is not correct.

> project (string) | empty | This variable is datastore project Id. If it is empty, GCLOUD_PROJECT environment variable will be used. Either you should set it directly or the environment variable should be set.

I will fix it. 
The `GCLOUD_PROJECT` is deprecated so I think  new description is

```text
 project (string) | test | This variable is datastore project Id.
```